nbs/15_gsa_gls/30-real_phenotypes/10-summarize-phenomexcan.ipynb

```diff
@@ -0,0 +1,2001 @@
+{
```

Comment:
<p>whoa~! lots of significant hits, is this expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I don't know if it is expected, but it's similar to other large runs on multiple phenotypes (4,091 here). There is another PR with the null simulations also, where we found that the model is approximately well calibrated and with mean type I errors around 0.05.</p><p> </p><p>Some of the LV-trait associations make a lot of sense, like LV246, with genes expressed in adipose tissue, and associated to many cardiovascular traits.</p>